File: API/APARTMENTS/Models/Apartment.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text.Json.Serialization;
using System.Threading.Tasks;

namespace APARTMENTS.Models
{
    public class Apartment
    {
        public int Id { get; set; }
        public string ApartmentDescription { get; set; }
        public int NumberOfRooms { get; set; }
        public int MonthlyPrice { get; set; }
        public ICollection<Adress> Adress { get; set; }
        public User Renter { get; set; }
        public int? RenterId { get; set; }
        public int OwnerId { get; set; }
        public User Owner { get; set; }
        public List<Photo> Photos { get; set; }
        public ICollection<Comment> Comments { get; set; }

    }
}

File: StanAngular2/client/src/app/_models/apartment.ts
import { Adress, IAdress } from "./adress";
import { Photo } from "./photo";

export interface Apartment{
    id: number;
    apartmentDescription: string;
    numberOfRooms: number;
    monthlyPrice: number;
    ownerId: number;
    renterId:number;
    adress : Adress[]
    photos : Photo[]
}
export class ApartmentClass{
    id: number= 0;
    apartmentDescription: string="";
    numberOfRooms: number=0;
    monthlyPrice: number=0;
    renterId: number = 0;
    ownerId: number=0;
}
export class RentApartmentClass{
    renterId:number=0;
    apartmentId: number=0;
    
}
export class UpApartmentClass{
    apartmentDescription: string="";
    numberOfRooms: number=0;
    monthlyPrice: number=0;}
